docs/CONTRIBUTING.rst

```diff
@@ -0,0 +1,107 @@
+.. _contributing-to-plasmapy:
```

Comment:
`/home/circleci/project/docs/CONTRIBUTING.rst: WARNING: document isn't included in any toctree`